ch04.adoc

```diff
@@ -185,97 +187,294 @@ The `computed_standard_name` attribute indicates that the values in variable
+
+The time zone _Z_ has four possible formats, any of which may be prefixed with a sign:
+
+** _H_, the hour alone, of one or two digits e.g. **`-6`**, **`2`**, **`+11`**, which is sufficient for many time zones.
+
+** __H__:__M__, where _H_ is hour and _M_ minute, each of one or two digits, e.g. **`5:30`**.
+
+** four digits, of which the first pair are the hours and the second the minutes e.g. **`0530`**.
+
+** three digits, of which the first is the hour (0--9) e.g. **`530`**.
+
+For example, **`seconds since 1992-10-8 15:15:42.5 -6:00`** indicates seconds since October 8th, 1992 at 3 hours, 15 minutes and 42.5 seconds in the afternoon, in the time zone which is centred on -90 `degrees_east` of longitude, six hours behind the default time zone.
+Subtracting the time zone from the date/time specified converts it to the default time zone e.g. **`1989-12-31 18:00:00 -6`** identifies the same instant as **`1990-1-1 0:0:0`**.
+
+
+[[calendar, Section 4.4.2, "Calendar"]]
 ==== Calendar
 
-A date/time is the set of numbers which together identify an instant of time, namely its year, month, day, hour, minute and second, where the second may have a fraction but the others are all integer.
-A time coordinate value represents a date/time.
-In order to calculate a time coordinate value from a date/time, or the reverse, one must know the **`units`** attribute of the time coordinate variable (containing the time unit of the coordinate values and the reference date/time) and the calendar.
+In order to calculate a time coordinate value from a date/time, or the reverse, one must know the **`units`** attribute of the time coordinate variable (containing the time unit of the coordinate values and the reference date/time, see <<time-coordinate-units>>) and the calendar, indicated by the **`calendar`** attribute.
 The choice of calendar defines the set of dates (year-month-day combinations) which are permitted, and therefore it specifies the number of days between the times of **`0:0:0`** (midnight) on any two dates.
-Date/times which are not permitted in a given calendar are prohibited in both the encoded time coordinate values, and in the reference date/time string.
-It is recommended that the calendar be specified by the **`calendar`** attribute of the time coordinate variable.
-
-When a time coordinate value is calculated from a date/time, or the reverse, it is assumed that the coordinate value increases by exactly 60 seconds from the start of any minute (identified by year, month, day, hour, minute, all being integers) to the start of the next minute, with no leap seconds, in all CF calendars.
-This assumption has various consequences when real-world date/times from calendars which do contain leap seconds (such as UTC) are stored in time coordinate variables:
+Date/times which are not permitted in a given calendar are prohibited both in the time coordinate values, and in the reference date/time string.
```

Comment:
Date/times which are not permitted in a given calendar are prohibited both in the time coordinate values and in the reference date/time string.

[removed the comma]

Reply:
Done